drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -135,6 +135,7 @@ removeCollectionWarningText=This action cannot be undone.
 #
 settings=Settings
 dmnPathErrorLabel=Impossible to find ''{0}'' file. Please select one.
+dmnPathErrorDetailedLabel=Impossible to find referred DMN file: ''{0}''. Please provide it and update setting panel.
```

Comment:
Could we please change it to be mor concrete on what should be an update
Ex: "Please provide it and link in Settings"

Reply:
@dupliaka Done!